dev/AutoSuggestBox/AutoSuggestBox_themeresources.xaml

```diff
@@ -112,7 +112,6 @@
                                                     VerticalAlignment="Center"
                                                     HorizontalAlignment="Center"
                                                     FontStyle="Normal"
-                                                    FontSize="{ThemeResource AutoSuggestBoxIconFontSize}"
```

Comment:
Why removing this?

Reply:
I think that this was overwriting the FontSize setting for the component, prevemting it from receiving the templates property "FontSize"

Reply:
Oh I see, thanks!